core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -136,6 +136,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     /// Border growth
     BorderGrowthPercentage("[relativeAmount]% Culture cost of natural border growth [cityFilter]", UniqueTarget.Global, UniqueTarget.FollowerBelief),
     TileCostPercentage("[relativeAmount]% Gold cost of acquiring tiles [cityFilter]", UniqueTarget.FollowerBelief, UniqueTarget.Global),
+    CanPurchaseOtherCivTiles("May purchase other Civs Tiles", UniqueTarget.FollowerBelief, UniqueTarget.Global),
```

Comment:
As a *follower belief* means that this needs to take the city unique into account, which isn't what's happening now

Reply:
> As a _follower belief_ means that this needs to take the city unique into account, which isn't what's happening now

Isn't the check for if a city can purchase a tile city based?

Reply:
The uncivInfo.hasUnique you wrote should check if the *city* has the unique